config/pod.yaml.hbs

```diff
@@ -14,6 +14,8 @@ spec:
   - name: build
     image: {{container}}
     imagePullPolicy: Always
+    securityContext:
+      privileged: true
```

Comment:
@parthasl please make this optional. By default for k8s executor we shouldn't turn this on, unless user needs it for DIND use cases.

Reply:
PRs raised -
https://github.com/screwdriver-cd/executor-k8s/pull/115
https://github.com/screwdriver-cd/buildcluster-queue-worker/pull/48